app/models/assignment_team.rb

```diff
@@ -16,6 +16,10 @@ def parent_model
     "Assignment"
   end
 
+  def self.parent_model_for_id (id)
```

Comment:
This is not necessary, please check this line on very top of AssignmentTeam model:
`belongs_to :assignment, class_name: 'Assignment', foreign_key: 'parent_id'`